### PR TITLE
Updating Visual Studio references

### PR DESCRIPTION
**Pull request by kraigb**
VS documentation has a dedicated page for its PyLint integration

### Fixes / new features
- Added VS reference in first list
- Added VS context menu command in second list.
- Removed obsolete content for VS 2003.

### CODE REVIEW

File: doc/user_guide/ide-integration.rst [hunk 1]
```diff
@@ -104,13 +107,6 @@ the following instead of a post-command-hook
       "Display the error in the mini-buffer rather than having to mouse over it"
       (show-fly-err-at-point))
 
-Setup the MS Visual Studio .NET 2003 editor to call Pylint
-==========================================================
-
-.. image:: _static/vs2003_config.jpeg
```
Comment (rogalski) [inline doc/user_guide/ide-integration.rst]: shouldn't we also get rid of `doc/_static/vs2003_config.jpeg`?

Comment (kraigb) [inline doc/user_guide/ide-integration.rst]: Yes, that image is no longer necessary.

Comment (kraigb) [inline doc/user_guide/ide-integration.rst]: I've added the deletion to the PR.

**Comment by PCManticore** [comment]
Thank you!

**Pull request merged**